e2e-tests/cypress/e2e/journeys/insurance/your-business/nature-of-business/nature-of-business-page.spec.js

```diff
@@ -0,0 +1,112 @@
+import { natureOfBusiness } from '../../../../pages/your-business';
+import partials from '../../../../partials';
+import { heading, submitButton, saveAndBackButton } from '../../../../pages/shared';
+import { PAGES, BUTTONS, LINKS } from '../../../../../../content-strings';
+import { EXPORTER_BUSINESS_FIELDS as FIELDS } from '../../../../../../content-strings/fields/insurance/exporter-business';
+import { ROUTES, FIELD_IDS } from '../../../../../../constants';
+import getReferenceNumber from '../../../../helpers/get-reference-number';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.EXPORTER_BUSINESS.NATURE_OF_YOUR_BUSINESS;
+
+const {
+  NATURE_OF_YOUR_BUSINESS: {
+    GOODS_OR_SERVICES,
+  },
+} = FIELD_IDS.INSURANCE.EXPORTER_BUSINESS;
+
+const insuranceStart = ROUTES.INSURANCE.START;
+
+context('Insurance - Your business - Nature of your business page - As an Exporter I want to enter the nature of my business So that UKEF can have clarity on the type of business that I do while processing my Export Insurance Application', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.visit(ROUTES.INSURANCE.START, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      const url = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.EXPORTER_BUSINESS.NATURE_OF_BUSINESS}`;
+
+      cy.visit(url, {
+        auth: {
+          username: Cypress.config('basicAuthKey'),
+          password: Cypress.config('basicAuthSecret'),
+        },
+      });
+
+      cy.url().should('eq', url);
+    });
+  });
```

Comment:
Could do this now or in the next PR but I think we should have the nested pages of a section following the actual user flow through the task list etc,

e.g in the second page of "policy and exports", created  a cypress command to complete each form we have this:

```js
cy.submitInsuranceEligibilityAndStartApplication();

taskList.prepareApplication.tasks.policyTypeAndExports.link().click();

cy.completeAndSubmitPolicyTypeForm();

cy.completeAndSubmitSingleContractPolicyForm();
```

Also helps us for future features, e.g with "check your answers" we can just call these commands in the check answers tests. And similar when we have the entire application built and we want to test submitted the application 😎

Reply:
another PR please